db-init.js

```diff
@@ -0,0 +1,35 @@
+import { DynamoDB } from "@aws-sdk/client-dynamodb"
+
+const { NODE_ENV, SETUP_DB=false } = process.env.NODE_ENV;
```

Comment:
If we destructure these variables from `process.env.NODE_ENV`, this would not work.